SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialTypebindingManager.cpp

```diff
@@ -260,6 +286,13 @@ bool USpatialTypebindingManager::FindOffsetByComponentId(Worker_ComponentId Comp
 		return true;
 	}
 
+	if (UClass* Class = FindClassByComponentId(ComponentId))
+	{
+		AddTypebindingsForClass(Class);
+		OutOffset = ComponentToOffsetMap[ComponentId];
```

Comment:
Worth adding a check here that the ComponentToOffsetMap entry exists

Reply:
It must exist, otherwise the implementation is broken. Should this be a `FindChecked` or I should stick to `Find` and null-check?